Formula/qbs.rb

```diff
@@ -15,7 +15,7 @@ class Qbs < Formula
   depends_on "qt"
 
   def install
-    system "qmake", "qbs.pro", "-r", "QBS_INSTALL_PREFIX=#{prefix}", "CONFIG+=qbs_disable_rpath"
+    system "qmake", "qbs.pro", "QBS_INSTALL_PREFIX=#{prefix}", "CONFIG+=qbs_disable_rpath"
     system "make", "install", "INSTALL_ROOT=/"
```

Comment:
To fix the build you can call system "make" before calling "make install". It appears, qmake generates invalid dependencies for install targets.

Reply:
Thanks for investigating! I've adjusted qbs.rb accordingly.